lms/templates/video.html

```diff
@@ -46,44 +48,29 @@ <h4 class="hd hd-4 video-hls-error is-hidden">
 
     <div class="focus_grabber last"></div>
 
-    % if (download_video_link or track or handout or branding_info or public_video_url) and not hide_downloads:
+    % if (download_video_link or track or handout or branding_info or public_sharing_enabled) and not hide_downloads:
     <h3 class="hd hd-4 downloads-heading sr" id="video-download-transcripts_${id}">${_('Downloads and transcripts')}</h3>
     <div class="wrapper-downloads" role="region" aria-labelledby="video-download-transcripts_${id}">
-        % if download_video_link or public_video_url:
+        % if download_video_link or public_sharing_enabled:
             <div class="wrapper-download-video">
                 <h4 class="hd hd-5">${_('Video')}</h4>
-                % if public_video_url:
-                    <span class="icon fa fa-share-alt" aria-hidden="true"></span>
-                    ${_('Share on:')}
-                    <a
-                        class="btn-link"
-                        id="twitter-share-button"
-                        href="${public_video_url}"
-                        target="_blank"
-                    >
-                        <span class="icon fa fa-twitter-square" aria-hidden="true"></span>
-                        <span class="sr">${_('Share on Twitter')}</span>
-                    </a>
-                    <a
-                        class="btn-link"
-                        id="facebook-share-button"
-                        href="${public_video_url}"
-                        target="_blank"
-                    >
-                        <span class="icon fa fa-facebook-square" aria-hidden="true"></span>
-                        <span class="sr">${_('Share on Facebook')}</span>
-                    </a>
-                    <a
-                        class="btn-link"
-                        id="linkedin-share-button"
-                        href="${public_video_url}"
-                        target="_blank"
```

Comment:
@nsprenkle See the behavior change here.

Reply:
Fixing in https://github.com/openedx/edx-platform/pull/32187